makefile

```diff
@@ -0,0 +1,83 @@
+.PHONY=all chebi-subsets
+
+# All ------------------------------------------------------------------------------------------------------------------
+all: chebi-subsets
+
+# Analysis -------------------------------------------------------------------------------------------------------------
+input/analysis/:
+	mkdir -p $@
+
+output/analysis/:
+	mkdir -p $@
+
+# - ChEBI subsets
+PART_MAPPINGS=loinc_release/Loinc_2.78/AccessoryFiles/PartFile/PartRelatedCodeMapping.csv
+CHEBI_OWL=input/analysis/chebi.owl
+CHEBI_MODULE=output/analysis/chebi_module.txt
+CHEBI_OUT_BOT=output/analysis/chebi-subset-BOT.owl
+CHEBI_OUT_MIREOT=output/analysis/chebi-subset-MIREOT.owl
+
+chebi-subsets: $(CHEBI_OUT_BOT) $(CHEBI_OUT_MIREOT)
+
+input/analysis/chebi.owl.gz: | input/analysis/
+	wget -O $@ ftp://ftp.ebi.ac.uk/pub/databases/chebi/ontology/chebi.owl.gz
```

Comment:
This is more a general thing but always use PURLs for ontology download locations

Reply:
@matentzn This is a good idea. Don't we typically use Bioregistry as the central location for getting canonical PURLs?

- https://bioregistry.io/registry/chebi

However, I don't see an `.owl` URI there, only URIs for prefix maps.

There is a URI for it at Ontobee:
- https://ontobee.org/ontology/CHEBI

These PURLs have the disadvantage though of pointing to the `.owl`, not the `.owl.gz`, which I think is a better option whenever it is available.

Reply:
No, we should always use official OBO PURLS: 

http://purl.obolibrary.org/obo/chebi.owl
http://purl.obolibrary.org/obo/chebi/chebi.gz.owl

etc..

Reply:
@matentzn OK that's good, I will use that.

But what I'm sort of asking is how do I know what the PURL URI is for a given ontology URI? Is there somewhere that I can search?

I thought that place was BioRegistry, but that's not the case here.

I suppose I could just type out http://purl.obolibrary.org/obo/MAIN_ONTOLOGY_SPELLING.owl or http://purl.obolibrary.org/obo/MAIN_ONTOLOGY_SPELLING/MAIN_ONTOLOGY_SPELLING.FILE_EXTENSION and check to see if they exist, but that's not a good UX.